client/api/PollingProcessor.cs

```diff
@@ -131,6 +133,21 @@ private async Task ProcessSegments()
                 throw;
             }
         }
+        
+        public void TriggerProcessSegments()
```

Comment:
This isn't used. It requires more discussion about how we should recover from invalid cache states. We'd be adding latency to evaluations that are critical.  
In the current issue we are troubleshooting, if it's determined that it is caused by invalid cache state, then we can call this from the variation methods to recover.